extractor/src/main/scala/com/digitalasset/extractor/writers/postgresql/Queries.scala

```diff
@@ -302,7 +302,7 @@ object Queries {
             toJsonString(l)
           )
         case V.ValueInt64(value) => Fragment("?", value)
-        case V.ValueDecimal(value) => Fragment("?::numeric(38,10)", value: BigDecimal)
+        case V.ValueNumeric(value) => Fragment("?::numeric(38,10)", value: BigDecimal)
```

Comment:
Seems like this SQL conversion won't be right anymore?

Reply:
Likewise for `MultiTableDataFormat#mapSQLType`.

Reply:
We still cannot create Numeric with a scale different from 10.

Anyway `MultiTableDataFormat#mapSQLType` needs an updated version of the interface reader which will come in a following PR. 

I add a comment to not forget the fixes